server.go

```diff
@@ -126,14 +132,28 @@ func (s *Server) matchingRoute(path string) (*Route, map[string]string) {
 }
 
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+
+	shutdownTracing, err := tracing.Instrument(s.tracingConfig, s.Logger)
```

Comment:
`http.ListenAndServe` should be blocking, could this be moved to our `ListenAndServe` function so we create only a single tracing instance?

Reply:
oooooh I couldn't wrap my head around that before but that makes sense 👍

Reply:
done in ad39aaab08c4679858074f592e61bdaa69573405